ocdaction/core/views.py

```diff
@@ -24,12 +28,19 @@ class AboutView(TemplateView):
     template_name = "core/about.html"
 
 
-class ActView(TemplateView):
+@login_required
+def ActView(request):
     """
     The Act View.
     """
     template_name = "core/act.html"
 
+    # Get the logged in user, then retrieve the users unarchived tasks
+    current_user = OCDActionUser.objects.get(id=request.user.id)
+    task_list = Task.objects.filter(user=current_user.id, is_archived=False)
```

Comment:
@lmash sorry I mean like this: `task_list = Task.objects.filter(user=request.user, is_archived=False)

And this line `current_user = OCDActionUser.objects.get(id=request.user.id)` is not needed